src/main/scala/cromwell/binding/types/WdlFileGlobType.scala

```diff
@@ -0,0 +1,33 @@
+package cromwell.binding.types
+
+import cromwell.binding.values.{WdlFile, WdlFileGlob, WdlString}
+import spray.json.JsString
+
+import scala.util.{Success, Try}
+
+// TODO: Do we even need this?
```

Comment:
Yeah - this is ONLY the local backend which doesn't ever need to run the FileEvaluator before running.

The WdlFileGlob was intended for when the JES backend is turning WdlFile's into JesOutputs.

It *may* still be unnecessary but we haven't hit the "do we need it" point yet.